autokeras/utils.py

```diff
@@ -104,3 +106,59 @@ def is_label(y):
         Boolean. Whether the targets are plain label, not encoded.
     """
     return len(y.flatten()) == len(y) and len(set(y.flatten())) > 2
+
+
+def pickle_from_file(path):
+    """Load the pickle file from the provided path and returns the object."""
+    return pickle.load(open(path, 'rb'))
+
+
+def pickle_to_file(obj, path):
+    """Save the pickle file to the specified path."""
+    pickle.dump(obj, open(path, 'wb'))
+
+
+class OneHotEncoder(object):
+    """A class that can format data.
+
+    This class provides ways to transform data's classification label into
+    vector.
+
+    # Arguments
+        data: The input data
+        num_classes: The number of classes in the classification problem.
+        labels: The number of labels.
+        label_to_vec: Mapping from label to vector.
+        int_to_label: Mapping from int to label.
+    """
+
+    def __init__(self):
+        """Initialize a OneHotEncoder"""
+        self.data = None
+        self.num_classes = 0
+        self.labels = None
+        self.label_to_vec = {}
+        self.int_to_label = {}
+
+    def fit(self, data):
+        """Create mapping from label to vector, and vector to label."""
```

Comment:
It would be useful to document assumptions about `data` (what types are supported?)